README.md

```diff
@@ -75,3 +80,22 @@ tests with the following command:
 ```bash
 make test-e2e
 ```
+
+### Running the end-to-end tests on ROSA STS cluster
+
+**Prerequisistes**:
+- The operator has to be deployed with [the prerequisites for the STS cluster](./docs/prerequisites.md#for-sts-clusters).
+- The controller's secret needs to be created as described in [the installation instructions for the STS cluster](./docs/install.md#post-operator-installation).
+- The test WAFv2 and WAF regional WebACLs need to be created. You can use the following commands:
+```bash
+aws wafv2 create-web-acl --name "echoserver-acl" --scope REGIONAL --default-action '{"Block":{}}'  --visibility-config '{"MetricName":"echoserver","CloudWatchMetricsEnabled": false,"SampledRequestsEnabled":false}'
+aws waf-regional create-web-acl --name "echoserverclassicacl" --metric-name "echoserverclassicacl" --default-action '{"Type":"BLOCK"}' --change-token "$(aws waf-regional get-change-token)"
+```
+
+Now you can run the e2e test with the following commands:
+```bash
+export ALBO_E2E_PLATFORM=ROSA
+export ALBO_E2E_WAF_WEBACL_ID=<wafregional-webacl-id>
+export ALBO_E2E_WAFV2_WEBACL_ARN=<wafv2-webacl-arn>
```

Comment:
Do you need to describe how to get the `wafregional-webacl-id` and `wafv2-webacl-arn`?  If they come from the two commands above, maybe mention that above.